apps-devstg/us-east-1/security-certs/config.tf

```diff
@@ -13,20 +15,22 @@ provider "aws" {
 # binbash-shared route53 cross-account ACM dns validation update
 #
 provider "aws" {
-  region                  = var.region
-  profile                 = var.profile_shared
-  shared_credentials_file = "~/.aws/${var.project}/config"
-  alias                   = "shared-route53"
+  region  = var.region
+  profile = "${var.project}-shared-devops"
+  # comment this 2 lines for aws sso enable
+  # shared_credentials_files = ["~/.aws/${var.project}/credentials"]
+  # shared_config_files      = ["~/.aws/${var.project}/config"]
```

Comment:
@rodriguez-matias Same comment as before, if this lines still exist as commented please let's remove them. Thanks!